### DISCUSSION TITLE
Show video info (views+date) in Playlists?

### DISCUSSION THREAD

**Discussion by luckkmaxx** [libre-tube/LibreTube#3869] [Polls]
In the playlists you only can see title, channel and duration of each video, but there is the only place we cannot see the number of views and release date...
Look this relevant for you?

**Comment by Bnyro** [comment]
That's not API supported, so it doesn't matter if you think it's relevant or not.

**Reply by luckkmaxx** [reply]
ok thank you. I thought it was removed or dismissed for a reason.

**Reply by Bnyro** [reply]
No, for instance YouTube in the web doesn't display the info either when playing a playlist.

**Reply by Bnyro** [reply]
I would agree that it'd be useful though.

**Reply by FireMasterK** [reply]
It could maybe be added in the future for Piped playlists once https://github.com/TeamPiped/Piped/issues/629 is done.

**Comment by SameenAhnaf** [comment]
Reopening as this feature is supported by Piped API
